train.py

```diff
@@ -161,12 +169,22 @@ def main(_argv):
         train_dataset = dataset.load_fake_dataset()
     train_dataset = train_dataset.shuffle(buffer_size=512)
     train_dataset = train_dataset.batch(FLAGS.batch_size)
+
+    # data augmentation just for training
+    augmentation = tf.keras.Sequential(
+        [
+            GaussianNoise(0.25),
```

Comment:
Yes surely, this can be added to configurable parameters but that change will be part of the ultrasound training issue in the medical imaging pipeline. I'll make it configurable in that repo using yaml config file.